File: 1115.py
a,b = map(int, input().split())
while a != 0 or b != 0:    
    if(a>0 and b>0):
        print("primeiro")
    elif(a>0 and b<0):
        print("quarto")
    elif(a<0 and b<0):
        print("terceiro")
    elif(a<0 and b>0):
        print("segundo")
    a,b = map(int, input().split())
    

File: 1115.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    int x,y;
    while(true){
        cin >> x >> y;
        if(x==0||y==0){
            break;
        }else{
            if(x>0 && y>0) cout << "primeiro\n";
            else if (x>0 && y<0) cout << "quarto\n";
            else if (x<0 && y<0) cout << "terceiro\n";
            else cout << "segundo\n";
            
        }
    }
}